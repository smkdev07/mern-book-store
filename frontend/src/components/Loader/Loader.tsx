import React from 'react';

import { Spinner } from 'react-bootstrap';

const styles = {
  display: 'block',
  height: '100px',
  width: '100px',
  margin: 'auto',
};

const Loader: React.FC = () => (
  <Spinner animation="border" role="status" variant="primary" style={styles}>
    <span className="sr-only">Loading...</span>
  </Spinner>
);

export default Loader;
