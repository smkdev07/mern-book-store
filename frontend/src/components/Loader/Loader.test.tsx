import React from 'react';
import { shallow } from 'enzyme';

import Loader from './Loader';

const setup = () => {
  return shallow(<Loader />);
};

describe('<Loader />', () => {
  it('renders the <Loader /> component', () => {
    const wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
