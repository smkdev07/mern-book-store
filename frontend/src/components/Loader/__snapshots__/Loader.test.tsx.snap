// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> renders the <Loader /> component 1`] = `ShallowWrapper {}`;
